### ISSUE TITLE
Make `error` module public

### ISSUE THREAD

**Issue by bitcapybara** [rustysec/fuzzyhash-rs#12]
Hi, Is it possible to make the error module public? In my program, I want to match `Error::IncompatibleBlockSizes` and skip it to ignore this match
Something like this:
```rust
for hash in hash_file.lines() {
    match FuzzyHash::compare(&input_hash, hash) {
        Ok(dist) if dist > 80 => {
            println!("matched, {}", hash)
        }
        Err(Error::IncompatibleBlockSizes) => {}
        Err(e) => Err(e)?,
        _ => continue,
    }
}
```

**Comment by rustysec** [comment]
yes :100:

**Comment by rustysec** [comment]
just published 0.2.2 to crates.io, let me know how that works out for you and we can close this issue.

**Comment by bitcapybara** [comment]
Thank you very much, this is very useful for me!
